TinCan/RemoteLRS.cs

```diff
@@ -101,6 +114,34 @@ public MyHttpResponse(HttpWebResponse webResp)
                     Content = ReadFully(stream, (int)webResp.ContentLength);
                 }
             }
+
+            public MyHttpResponse(HttpResponseMessage httpResponseMessage)
+            
+            {
+                Status = httpResponseMessage.StatusCode;
+                try
+                {
+                    if (httpResponseMessage.Headers.Contains("Etag"))
+                    {
+                        Etag = httpResponseMessage.Headers.GetValues("Etag")?.FirstOrDefault()?.ToString();
+                    }
+
+                    if (httpResponseMessage.Content.Headers.TryGetValues("LastModified", out var values))
+                    {
+                        // Get the Last-Modified header value and parse it to DateTime
+                        var lastModifiedString = values.ToString();
+
+                        if (DateTime.TryParse(lastModifiedString, out DateTime lastModifiedDate))
+                        {
+                            LastModified = lastModifiedDate;
+                        }
+                    }
+                }
+                catch
+                {
+                    //sometimes will throw an exception, just ignore

```

Comment:
Why? Can't we at-least log the exception? Or are we retrying here?

Reply:
Existing code was not doing anything, so I am not doing anything. it may change the behavior

Reply:
Xapi service will retry in our case